executor/ddl.go

```diff
@@ -250,8 +250,11 @@ func (e *DDLExec) executeDropTable(s *ast.DropTableStmt) error {
 			return errors.Trace(err)
 		}
 	}
-	if len(notExistTables) > 0 && !s.IfExists {
-		return infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ","))
+	if len(notExistTables) > 0 {
+		if !s.IfExists {
+			return infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ","))
+		}
+		e.ctx.GetSessionVars().StmtCtx.AppendNote(infoschema.ErrTableDropExists.GenWithStackByArgs(strings.Join(notExistTables, ",")))
```

Comment:
mysql:
```
mysql root@127.0.0.1:test> drop table if exists t1,t2;
Query OK, 0 rows affected
mysql root@127.0.0.1:test> show warnings
+-------+------+-------------------------+
| Level | Code | Message                 |
+-------+------+-------------------------+
| Note  | 1051 | Unknown table 'test.t1' |
| Note  | 1051 | Unknown table 'test.t2' |
+-------+------+-------------------------+
```
tidb:
```
mysql root@127.0.0.1:test> drop table if exists t1,t2;
Query OK, 0 rows affected
mysql root@127.0.0.1:test> show warnings
+-------+------+---------------------------------+
| Level | Code | Message                         |
+-------+------+---------------------------------+
| Note  | 1051 | Unknown table 'test.t1,test.t2' |
+-------+------+---------------------------------+
```
2 warning msg in mysql, 1 warning in tidb. Both ok for me.
 @morgo PTAL

Reply:
@crazycs520 Good catch.  I think we should aim for compatibility here, unfortunately.  Consider the case that someone tries to drop 10 tables and wants to know which ones existed prior.  They would have to have special handling for TiDB.

Reply:
@crazycs520 we should keep compatibility with MySQL